src/view/forms/organization/Organization.tsx

```diff
@@ -121,3 +148,97 @@ export const CreateOrganizationForm = ({
 
   return getCurrentForm();
 };
+
+/**
+ * One-part form used to build an `UpdateOrganizationAction` payload object
+ */
+export const UpdateOrganizationForm = ({
+  existingOrg,
+  setBatchStatusLink,
+}: OrgTransactionFormProps) => {
+  const { signer } = useAuth();
+  const [errMsg, setErrMsg] = useState('');
+  const [org, setOrg] = useState<IUpdateOrganizationActionStrict>({
+    contacts: existingOrg.contacts as IContactStrict[],
```

Comment:
I believe this can just be
```suggestion
    contacts: existingOrg.contacts,
```

I didn't get any TS errors.

Reply:
I did get TS errors